style/DesktopIndex.css

```diff
@@ -0,0 +1,395 @@
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  * {
+    box-sizing: border-box;
+    scroll-behavior: smooth;
+  }
+
+  body {
+    font-family: "Lato", 'Verdana', 'Geneva', Tahoma, sans-serif;
+    font-style: normal;
+    width: 100%;
+    padding: 0;
+    margin: auto;
+  }
+
+  .top-nav {
+    display: flex;
+    flex-direction: row;
+    gap: 5px;
+    justify-content: right;
+    justify-items: center;
+    align-items: center;
+    text-decoration: none;
+    background-color: rgb(21, 1, 1);
+  }
+
+  nav {
+    display: flex;
+    width: 100%;
+    padding-right: 40px;
+    padding-left: 60px;
+  }
+
+  .top-nav-1 {
+    display: flex;
+    justify-items: center;
+    padding: 10px 20px;
+    text-decoration: none;
+  }
+
+  .top-nav-1 a {
+    display: flex;
+    justify-items: center;
+    padding: 5px;
+    text-decoration: none;
+    color: #fff;
+  }
+
+  .top-nav-2 {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 20px;
+    padding: 10px 20px;
+  }
+
+  .top-nav-2 a {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  #header-logo {
+    display: flex;
+    width: 100%;
+    align-self: center;
+    margin-left: 0;
+    padding-left: 20px;
+  }
+
+  .header-logo-image {
+    display: flex;
+    width: 20%;
+  }
+
+  .about-contact p {
+    display: flex;
+    justify-content: center;
+    width: 80%;
+    margin: auto;
+    font-size: 14px;
+  }
+
+  .about-desc-1 p {
+    justify-content: center;
+    width: 100%;
+    margin: auto;
+    font-size: 18px;
+  }
+
+  .about-desc-2 p {
+    font-size: 13px;
+  }
+
+  .main-date p {
+    font-size: 25px;
+    width: 100%;
+  }
+
+  #header-logo p {
+    display: flex;
+    text-decoration: none;
+    align-self: center;
+  }
+
+  #nav-items {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-self: center;
+    gap: 20px;
+    font-family: "Inter", 'Cambria', 'Cochin', Georgia, Times, 'Times New Roman', serif;
+    font-style: normal;
+    font-size: 0.9375em;
+    width: 100%;
+    padding: 0 20px;
+  }
+
+  .nav-items a {
+    display: flex;
+    flex-direction: row;
+    gap: 50px;
+    font-family: "Inter", 'Cambria', 'Cochin', Georgia, Times, 'Times New Roman', serif;
+    font-style: normal;
+    font-size: 0.9375em;
+    text-decoration: none;
+  }
+
+  .see-whole-program a {
+    color: white;
+  }
+
+  .nav-items a:hover {
+    color: red;
+  }
+
+  #campaign {
+    border: 4px solid red;
+    padding: 6px;
+  }
+
+  .menu-btn {
+    display: none;
+  }
+
+  .main {
+    padding-top: 90px;
+    background-image: url('/images/main-bg.jpeg');
+  }
+
+  .main-container {
+    display: flex;
+    flex-direction: column;
+    gap: 13px;
+    width: 80%;
+    margin: auto;
+  }
```

Comment:
- [ ] Please add the correct background color which matches or looks almost like the design given. If you can't find the background image that looks great, I would suggest you use the `light-grey` background color
**Your design**
![image](https://user-images.githubusercontent.com/90222110/192013625-ce962248-d236-4247-9eda-2fb4a60adace.png)

Given design
![image](https://user-images.githubusercontent.com/90222110/192013690-b299af28-364b-40ec-b738-a911d009217d.png)